lib/shroud/plugin.rb

```diff
@@ -62,14 +62,18 @@ class DangerShroud < Plugin
     #          defines the required percentage of files modified in a PR for a passing build.
     #          default 90.
     #
-    # @param   [Boolean] failIfUnderThreshold
-    #          if true, will fail builds that are under the provided thresholds. if false, will only warn.
+    # @param   [Boolean] failIfUnderProjectThreshold
+    #          if true, will fail builds that are under the provided thresholds for the overall project. If false, will only warn.
+    #          default true.
+    #
+    # @param   [Boolean] failIfUnderFileThreshold
+    #          if true, will fail builds that are under the provided thresholds per file. If false, will only warn.
     #          default true.
     #
     # @return  [void]
-    def report(file, totalProjectThreshold = 90, modifiedFileThreshold = 90, failIfUnderThreshold = true)
+    def report(file, totalProjectThreshold = 90, modifiedFileThreshold = 90, failIfUnderProjectThreshold = true, failIfUnderFileThreshold = true)
```

Comment:
Does ruby let you do something like this:
```ruby
def report(file, totalProjectThreshold = 90, modifiedFileThreshold = 90, failIfUnderProjectThreshold = true, failIfUnderFileThreshold = failIfUnderProjectThreshold)
```
That would make it identical to the previous functionality. If not, this all seems great!